reana/reana_dev/wiki.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2020 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""`reana-dev`'s wiki commands."""
+
+import click
+
+from reana.config import REANA_LIST_DEMO_ALL
+
+GITHUB_REANAHUB_URL = "https://github.com/reanahub"
+CODECOV_REANAHUB_URL = "https://codecov.io/gh/reanahub"
+MAINT_BRANCHES = ["maint-0.7"]
+
+
+@click.group()
+def wiki_commands():
+    """Wiki commands group."""
+
+
+@wiki_commands.command(name="wiki-create-build-status-page")
+def create_build_status_page():
+    """Generate Markdown for the Build Status wiki page."""
+    sections = {
+        "researchers": {
+            "title": "For researchers",
+            "description": "Find out how you can use REANA to describe, run, preserve and reuse your analyses.",
+            "packages": {
+                "reana-client": {},
+                "docs.reana.io": {"coverage": False, "docs": False},
+                "www.reana.io": {"coverage": False, "docs": False},
+            },
+        },
+        "administrators": {
+            "title": "For administrators",
+            "description": "Install and manage the REANA reusable analysis platform on your own compute cloud.",
+            "packages": {
+                "reana-commons": {},
+                "reana-db": {},
+                "reana-job-controller": {},
+                "reana-message-broker": {},
+                "reana-server": {},
+                "reana-ui": {},
+                "reana-workflow-controller": {},
+                "reana-workflow-engine-cwl": {},
+                "reana-workflow-engine-serial": {},
+                "reana-workflow-engine-yadage": {},
+                "reana-workflow-monitor": {},
+            },
+        },
+        "developers": {
+            "title": "For developers",
+            "description": "Understand REANA source code, adapt it to your needs, contribute changes back.",
+            "packages": {"reana": {}, "pytest-reana": {}},
+        },
+        "environments": {
+            "simple": True,
+            "title": "Environments",
+            "description": "Selected containerised environments.",
+            "packages": {
+                "reana-env-aliphysics": {},
+                "reana-env-jupyter": {},
+                "reana-env-root6": {},
+            },
+        },
+        "authentication": {
+            "simple": True,
+            "title": "Authentication",
+            "description": "Selected authentication environments.",
+            "packages": {"reana-auth-krb5": {}, "reana-auth-vomsproxy": {},},
+        },
+        "examples": {
+            "simple": True,
+            "title": "Examples",
+            "description": "Selected reusable analysis examples.",
+            "packages": {demo: {} for demo in REANA_LIST_DEMO_ALL},
```

Comment:
Perhaps we need to `.sort()` them so that demo repositories are coming up alphabetically?  (thinking about that now the variable is a join of two lists)